packages/aws-cdk-lib/aws-ec2/lib/machine-image/machine-image.ts

```diff
@@ -652,11 +667,16 @@ export class GenericWindowsImage implements IMachineImage {
  * will be used on future runs. To refresh the AMI lookup, you will have to
  * evict the value from the cache using the `cdk context` command. See
  * https://docs.aws.amazon.com/cdk/latest/guide/context.html for more information.
+ * If `props.additionalCacheKey` is true, the context key use that value as a discriminator
+ * rather than the cached value being global across all lookups
```

Comment:
Other similar docstrings have "so that separate lookups with the same parameters can have separate cache lifecycles" - should we just remove that extra language here, use the same language as elsewhere here, or do you think it needs to be removed elsewhere?